### ISSUE TITLE
6.5 Opcodes?

### ISSUE THREAD

**Issue by Crusaddy** [Soreepeong/XivAlexander#658]
I am desperate and want to play with my friends but certified dummy

**Comment by v0di** [comment]
It seems they have just been updated. You can click [Configure] -> [Check for Opcode Updates]

**Comment by zerracas** [comment]
I agree, it says opcodes are updated but the animation lock (no clip) feature doesn't work at all, which makes me think they're not actually up to date.

**Comment by luke0717** [comment]
Definitely not updated yet, there's been no change to the build's opcode
